### PR TITLE
Added price and currencySymbol to loadProducts response

### PR DESCRIPTION
**Pull request by hedblad**
Provide price as a number, since it's hard to parse the priceString (i.e. when presenting a yearly subscription price as the price per month). Also added currencySymbol for convenience.

### CODE REVIEW

**Comment by chirag04** [comment]
Looks good. Thanks :+1: 

want to document all the available product keys in readme?  Somewhere in the buy product section?

Quick question: Are you using this for subscription?

**Comment by hedblad** [comment]
I can add it do the documentation next week, no problem. Might add a currencyCode as well, which can be used with Number.prototype.toLocaleString() in the js layer as well.

We haven't tested purchasing subscriptions yet, but I'll try soon. I am able to fetch subscription products using loadProducts.

**Comment by chirag04** [comment]
Let me know how it goes and keep sending those patches :+1:
